### ISSUE TITLE
Feature request: change size of 'abstract' panel...

### ISSUE THREAD

**Issue by nickschurch** [codeforscience/sciencefair#106]
I'd really like to be able to change the size of the panel that shows the abstract of the paper that's selected. That'll make it more useful for quickly reading an abstract before deciding to add it to a collection.

**Comment by blahah** [comment]
@nickschurch so, make the whole detail pane at the bottom resizeable, or have the paper details pop out into something more readable?

**Comment by nickschurch** [comment]
Yeah, Make the whole panel at the bottom resizable. I'm not a fan of pop-outs, but having an option to detatcht he panel completely might be nice too - that sounds like hard(er) work though.
